src/backend/api/meals.js

```diff
@@ -1,14 +1,70 @@
+const knex = require("knex");
 const express = require("express");
-const router = express.Router();
-const knex = require("../database");
+const database = require("../database");
+const { router } = require("../app");
```

Comment:
You should create a new `express.Router()` in this file, and then export it at the end. Moreover, all the paths should start from AFTER `/api/meals`, meaning the GET method for all meals should be `mealsRouter.get("/", ...)`.